assets/js/src/core/modules/data-object/editor/toolbar/toolbar.tsx

```diff
@@ -116,28 +95,63 @@ export const Toolbar = (): React.JSX.Element => {
         return propertyApi
       })
 
-      update.properties = propertyUpdate?.filter((property) => !property.inherited)
+      updatedData.properties = propertyUpdate?.filter((property) => !property.inherited)
     }
 
     const editableData = getModifiedDataObjectAttributes()
 
     if (Object.keys(editableData).length > 0) {
-      update.editableData = editableData
+      updatedData.editableData = editableData
     }
 
     const saveDataObjectPromise = saveDataObject({
       id,
       body: {
         data: {
-          ...update
+          ...updatedData
         }
       }
     })
 
     const saveSchedulesPromise = saveSchedules()
 
-    Promise.all([saveDataObjectPromise, saveSchedulesPromise]).catch((error) => {
+    Promise.all([saveDataObjectPromise, saveSchedulesPromise]).then(() => {
+      const invalidateVersionsList = invalidatingTags.ASSET_VERSIONS(id)
+
+      dispatch(api.util.invalidateTags(invalidateVersionsList))
```

Comment:
We should not do this here but implement it directly in the RTK query tags logic. Let's check it together with @xIrusux as she implemented the same approaches also for assets/other tabs.